### PR TITLE
Router added

### PR DESCRIPTION
**Pull request by temvvlena**
[](url)

Fixed Aux file

### CODE REVIEW

File: web/src/axios-orders.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const instance = axios.create({
+    baseURL: 'https://referral-3ffeb.firebaseio.com/'
```
Comment (ulziibay) [inline web/src/axios-orders.js:4]: Do you get the form data at this endpoint?

File: web/src/components/Layout/Layout.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import Aux from '../../hoc/Auxiliary';
+import classes from './Layout.css';
+import Toolbar from '../Navigation/Toolbar/Toolbar';
+import SideDrawer from '../Navigation/SideDrawer/SideDrawer';
+
+const layout = ( props ) => (
+    <Aux>
```
Comment (ulziibay) [inline web/src/components/Layout/Layout.js:9]: Do you still need the Aux?

File: web/src/components/ParticipantForm/Participant.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
+/* Mobile Layout */
```
Comment (ulziibay) [inline web/src/components/ParticipantForm/Participant.css:1]: Can you add a screenshot of how it looks on mobile?

File: web/src/components/ParticipantForm/ParticipantForm.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import classes from './Participant.css';
+import PFS from './ParticipantFormStates/PFS';
+import {Route} from 'react-router-dom';
+
+// import Spinner from '../../components/UI/Spinner/Spinner';
+// import axios from '../../axios-orders';
```
Comment (ulziibay) [inline web/src/components/ParticipantForm/ParticipantForm.js:7]: Please remove them if they are not used

File: web/src/components/ParticipantForm/ParticipantFormStates/PFS.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+                    <InputGroup id="text-input" placeholder="Placeholder text" />
+                </FormGroup>
+                <FormGroup
+                    label="Hello! It's nice to e-meet you. What's your name? (First name, Last name)"
+                    labelFor="text-input"
+                    labelInfo="(required)"
+                    inline={true}>
+                    <InputGroup id="text-input" placeholder="Placeholder text" />
+                </FormGroup>
+                <button clicked={this.submitHandler}>Submit</button>
```
Comment (ulziibay) [inline web/src/components/ParticipantForm/ParticipantFormStates/PFS.js:46]: Can we make this button the same as the button on the Referrer form?
Screen Shot 2020-07-21 at 10 34 55 AM
Screen Shot 2020-07-21 at 10 34 38 AM

File: web/src/components/ParticipantForm/ParticipantFormStates/PFS.js [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+            </div>
+        );
+    }
+}
+export default PFS
+
+//         let aForm = null;
+//         switch (this.props.type) {
```
Comment (ulziibay) [inline web/src/components/ParticipantForm/ParticipantFormStates/PFS.js:54]: delete this all?

File: web/src/containers/Forms/mainFormStructure.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+class FormBuilder extends Component {
+    render () {
+        return(
+            <div>
+                <Aux>
+                    <Switch>
+                        <Route path="/referrer" component={Referrer} />
+                        <Route path="/" component={Participant} />
+                    </Switch>
```
Comment (ulziibay) [inline web/src/containers/Forms/mainFormStructure.js:16]: Perhaps we can add the Admin tab with simple hello for now!

**Review by ulziibay — COMMENTED** [review]
looks good! couple of comments

File: web/src/axios-orders.js (hunk 1 above)
Comment (temvvlena) [reply web/src/axios-orders.js:4]: Yes. But, I used Firebase. The current baseURL link will be replaced by Postman Link.

File: web/src/components/Layout/Layout.js (hunk 2 above)
Comment (temvvlena) [reply web/src/components/Layout/Layout.js:9]: Yes. Otherwise, it wont run

File: web/src/components/ParticipantForm/Participant.css (hunk 3 above)
Comment (temvvlena) [reply web/src/components/ParticipantForm/Participant.css:1]: Sure! But, mobile version is incomplete. That will be one of my next issues to solve.

File: web/src/components/ParticipantForm/ParticipantForm.js (hunk 4 above)
Comment (temvvlena) [reply web/src/components/ParticipantForm/ParticipantForm.js:7]: Removed it

File: web/src/components/ParticipantForm/ParticipantFormStates/PFS.js (hunk 5 above)
Comment (temvvlena) [reply web/src/components/ParticipantForm/ParticipantFormStates/PFS.js:46]: Changed it

File: web/src/components/ParticipantForm/ParticipantFormStates/PFS.js (hunk 6 above)
Comment (temvvlena) [reply web/src/components/ParticipantForm/ParticipantFormStates/PFS.js:54]: deleted

File: web/src/containers/Forms/mainFormStructure.js (hunk 7 above)
Comment (temvvlena) [reply web/src/containers/Forms/mainFormStructure.js:16]: Okay, successfully finished that part

**Comment by temvvlena** [comment]
Firebase ee boliod, Eegii ahiin ugsun Database ee asaaj uzeh ni minii daraagiin task baih baih gej bodoj baina

**Pull request merged**